### ISSUE TITLE
Allow users to bookmark announcements and view them from a top-level nav link

### ISSUE THREAD

**Issue by WilHall** [thoughtbot/constable#656]
Migrated from [constable card](https://trello.com/c/JAMNx0dZ/61-allow-users-to-bookmark-announcements-list-them-as-another-top-level-link)

**Comment by tysongach** [comment]
It’d be great to get more detail in this issue, perhaps a job-to-be-done story. It’s currently describing a solution and doesn’t clarify what problem we’re hoping to solve.

@aledustet It looks like you originally added the Trello card - can you expand on this?

**Comment by aledustet** [comment]
Happy to 🤗 

This started as I keep referencing conversations in numerous constable posts and was keeping a list of useful or "I want to get back to these posts" a few weeks ago I saw german's announcement about  having a top-level tab for your [announcements](https://constable.io/announcements/2122-constable-update-see-your-and-other-user-s-announcements) 

Since we have a tab for the announcements we make, I thought we already have the template for a list of custom announcements, why can't we add one for bookmarks. This would be useful if you don't want all posts from a particular interest and just a particular one.

Hope this helps, very happy to discuss further.

I'll add this to the trello card

**Comment by tysongach** [comment]
> Since we have a tab for the announcements we make, I thought we already have the template for a list of custom announcements, why can't we add one for bookmarks.

Yes, but I imagine “bookmarks” are different because we’d need to provide facilities for adding them (and I assume removing, too). I think that is incrementally more complex than the “Your Announcements” tab.

> and was keeping a list of useful or "I want to get back to these posts"

Do browser bookmarks, or “read-later” services like Instapaper and Pocket not suffice to solve this need?
